tests/uptane_vector_tests.cc

```diff
@@ -15,178 +15,152 @@
 #include "storage/invstorage.h"
 #include "utilities/utils.h"
 
+std::string address;
+
+bool match_error(const Json::Value& error, const Uptane::Exception& e) {
+  if (error["director"]["update"]["err_msg"].asString() == e.what() ||
+      error["director"]["targets"][e.getName()]["err_msg"].asString() == e.what() ||
+      error["image_repo"]["update"]["err_msg"].asString() == e.what() ||
+      error["image_repo"]["targets"][e.getName()]["err_msg"].asString() == e.what()) {
```

Comment:
Perhaps, it makes sense to introduce a new class-wrapper that would wrap the vector's json and introduce methods required by its clients. As result of it, business logic of accessing and interpreting the vector's json fields (including comparison with a given exception) will be encapsulated in one place in contrast to spreading it throughout the code base in each vector client.

Reply:
Not a bad suggestion. As a first step, I moved all the parts that actually parse the json into a new class, which I think goes most of the way towards cleaning that up. I think getting more fine-grained is probably not necessary at this time.

Reply:
ok.